KMaina/Fast-Food-App
# #160298232 Page to Edit a Food Item

#### What does this PR do?
Allows an admin to edit a food item
#### Description of Task to be completed?
An admin should be able to

- Change the image of a food item

- Change he description of a food item

#### How should this be manually tested?

- Clone the repo onto your local machine.

- Open a terminal and run `cd Fast-Food-App`

- run `git checkout -b ft-admin-edit-add-160298232`

- run `git pull https://github.com/KMaina/Fast-Food-App.git ft-admin-edit-add-160298232`

- In the folder where you cloned the repo, you should open the UI folder

- Open the admin-edit-food.html to see it in your browser for review

#### Any background context you want to provide?
This is simply the UI design, no functionality has been built in yet.
#### What are the relevant pivotal tracker stories?
#160298232
